### ISSUE TITLE
Duplicate attempts blocks providing feedback, stranding assignment as Waiting for feedback

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1301]
**Description**
If an examiner by mistake grants more than one new attempt and then provides feedback, the student group is stranded at waiting for feedback on the last attempt given after the deadline expires while the examiner [is prevented from given further feedback since «Feedback is already published!»](https://github.com/devilry/devilry-django/blob/6.2.4/devilry/devilry_group/views/examiner/feedbackfeed_examiner.py#L227-L228).

My guess is that the first feedback provided is given to the oldest open attempt, but that the `save_object()` if-else looks at something else.

**Workaround**
Using Simple bulk feedback an examiner is able to give the student group a grade, and that will also change the status from Waiting for feedback to Corrected.

**Comment by Levijatan** [comment]
Yup the feedback was given to the oldest attempt. Could not find an east way to make sure that the feedback gets to the newest attempt, so ended up with changing how new attempts are made to check if there is no feedback published it moves the deadline rather then creating a new attempt. Commited in 326a5d4
